shell/server/sandcats.js

```diff
@@ -63,21 +63,24 @@ const pingUdp = () => {
     }
   });
 
-  socket.send(message, 0, message.length, 8080, SANDCATS_HOSTNAME, (err) => {
-    if (err) {
-      console.error("Couldn't send UDP sandcats ping", err);
-    }
-  });
+  socket.bind({ address: process.env.BIND_IP }, () => {
```

Comment:
Hmm, how are errors reported here? E.g. if the bind IP isn't a local address. Do they go to the callback, or does this throw? It's weird to me that bind() even takes a callback since the underlying syscall does not block anyway.